docs/build/wasm/contract_environment/chain-extension/chain_extensions.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 1
+---
+
+# Chain Extensions
+
+Chain extension is a way to extend contracts API to add contracts to runtime pallet interaction. By default, contracts can only do cross-contract calls within their environment (pallet-contracts). Chain extension allows to add custom callable pallet functions.
+
+![ink-ce](../../img/ink-ce.png)
+
+### What chain extensions are available ?
+
+#### XVM
+
+This chain extension enables usage of XVM in your contracts. More info in the [ink! XVM SDK repo](https://github.com/AstarNetwork/ink-xvm-sdk).
+
+#### Dapps Staking
+
+This chain extension adds call to `pallet_dapps_staking` so that you can use dApps Staking in your contracts. More info in the [chain-extensions contracts repo](https://github.com/swanky-dapps/chain-extension-contracts).
```

Comment:
```suggestion
This chain extension adds call to `pallet_dapps_staking` so that you can use dApp Staking in your contracts. More info in the [chain-extensions contracts repo](https://github.com/swanky-dapps/chain-extension-contracts).
```